packages/docs-site/src/components/presenters/text-card/index.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
```

Comment:
Think it's time we agree the file/folder naming convention to keep it consistent in all the places it needs to be

Reply:
Sounds good - I have just continued with what was previously being used in this codebase.